tensorflow/workspace.bzl

```diff
@@ -76,7 +76,6 @@ def tf_workspace(path_prefix = "", tf_repo_name = ""):
   native.new_http_archive(
       name = "libxsmm_archive",
       urls = [
-          # "http://bazel-mirror.storage.googleapis.com/github.com/hfp/libxsmm/archive/1.6.1.tar.gz",
```

Comment:
Let's also add the bazel-mirror one. The link should look like:
http://bazel-mirror.storage.googleapis.com/github.com/hfp/libxsmm/archive/1.6.5.tar.gz

@jart once we make this change, is there a mechanism to automatically mirror the file under bazel-mirror?